src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import cn from 'classnames';
+import { SearchLink } from '../SearchLink';
+import { SexFilter } from '../../types/SexFilter';
+
+type Props = {
```

Comment:
Similar names are okay for a small project, but on large projects we will have many interfaces and types, so it is better to give more detailed names, for example `PeopleFiltersProps`.
This applies to all code.